src/wazuh_db/schema_upgrade_v11.sql

```diff
@@ -0,0 +1,55 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Aug 14, 2023
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+CREATE TABLE IF NOT EXISTS _sys_programs (
+    scan_id INTEGER,
+    scan_time TEXT,
+    format TEXT,
+    name TEXT,
+    priority TEXT,
+    section TEXT,
+    size INTEGER CHECK (size >= 0),
+    vendor TEXT,
+    install_time TEXT,
+    version TEXT,
+    architecture TEXT,
+    multiarch TEXT,
+    source TEXT,
+    description TEXT,
+    location TEXT,
+    triaged INTEGER(1),
+    cpe TEXT,
+    msu_name TEXT,
+    checksum TEXT NOT NULL CHECK (checksum <> ''),
+    item_id TEXT,
+    PRIMARY KEY (scan_id, name, version, architecture, format, location)
+);
+
+INSERT INTO _sys_programs
+    SELECT scan_id, scan_time, CASE WHEN format IS NOT NULL THEN format ELSE '' END AS format,
+                   name, priority, section, size, vendor, install_time, version, architecture, multiarch, source, description,
+                   CASE WHEN location IS NOT NULL THEN location ELSE '' END AS location, triaged, cpe, msu_name,
```

Comment:
"location" field in the multiple package wrappers has "UNKNOW_VALUE" (" ") as the default value.
Shouldn't be " " value be set here also as default?

Reply:
Using the location as " " will lead to a mismatch in the checksum field for agents prior to 4.5.2 where that field is NULL in many cases and will affect synchronization.